tests/sandbox/headers/test_cors.py

```diff
@@ -25,7 +34,17 @@ def test_cors_options(nhsd_apim_proxy_url, method):
 @pytest.mark.parametrize("method", TEST_METHODS)
 def test_cors(nhsd_apim_proxy_url, method):
     """
-    .. py:function:: Test CORS response
+    .. py:function:: Scenario: A user submitting a request with an 'Origin' header receives \
+    an origin header in response
+
+        | **Given** the user provides an Origin header
+        | **When** the user submits a request
+        | **Then** the response contains the Origin header
+
+    **Asserts**
+    - Response contains 'Origin' header matching the provided value
```

Comment:
I believe we also assert against other headers being set in the response, worth mentioning here